lib/models/host.rb

```diff
@@ -119,7 +120,8 @@ def api_format
   end
 
   def define_cpus
-    cpus.map { |e| e[:cores] = cpu_cores }
+    total_cores = (cpu_cores/cpus.count)
```

Comment:
What would happen if cpus.count is null or 0, maybe we should consider those escenarios